webapp/first_snap/content/java/package.yaml

```diff
@@ -0,0 +1,69 @@
+    - code: |
+        name: freeplane
+    - warning: |
+        <span class="p-notification__status">Snap names are globally unique.</span> You should change any instance of <code>freeplane</code> to <code>freeplane-{name}</code> (where <code>{name}</code> is your name) before continuing to avoid conflicting with the official freeplane snap.
+    - text: Versions carry no semantic meaning in snaps.
+    - code: |
+        version: '1.6.10' 
+    - text: The summary can not exceed 79 characters. You can use a pipe ‘|’ in the description key to declare a multi-line description.
+    - code: |
+        description: |
+  security:
+    - text: The next section describes the level of confinement applied to your app.
+    - code: |
+        confinement: devmode
+    - text: Snaps are containerised to ensure more predictable application behaviour and greater security.
+    - text: It’s best to start a snap with the confinement in warning mode, rather than strictly applied. This is indicated through the <code>devmode</code> keyword.
+    - text: Once an app is working well in devmode, you can review confinement violations, add appropriate interfaces, and switch to strict confinement.
+  parts:
+    - text: |
+        Parts define how to build your app. Parts can be anything: programs, libraries, or other assets needed to create and run your application. Parts canpoint to local directories, remote git repositories, or tarballs. 
+    - code: |
+        parts:
+          freeplane:
+            after: [desktop-glib-only]
+            plugin: gradle
+            source: .
+            build: |
+              export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"
+              gradle release -x test -x createGitTag
+            install: |
+              unzip DIST/freeplane_bin-*.zip -d $SNAPCRAFT_PART_INSTALL/
+            build-packages:
+              - unzip
+              - openjdk-8-jdk
+    - text: |
+        The gradle plugin can build the application using standard parameters. In this case however we have overridden some gradle parameters (to disable testing, and prevent a new git tag being created), and set an appropriate JAVA_HOME in a build: script snippet.
+    - text: |
+        In the install: script snippet we’re unpacking the built application into a directory which later gets incorporated into the final snap, defined by the $SNAPCRAFT_PART_INSTALL variable.
+    - text: |    
+        The build requires an appropriate JDK/JRE and the install step requires the addition of the unzip command, so these are specified as build-packages.
+  apps:
+    - text: |
+        Apps are the commands and services exposed to end users. If your command name matches the snap name, users will be able run the command directly.
+    - text: |
+        If you don’t want your command prefixed you can request an alias for it on the Snapcraft forum. These are set up automatically when your snap is installed from the Snap Store.

```

Comment:
I would drop this entire line about prefixing apps. It's too much information for an introduction to snapping.